usecases/rule_snoozes.go

```diff
@@ -315,3 +345,13 @@ func (usecase RuleSnoozeUsecase) ActiveSnoozesForScenarioIteration(ctx context.C
 		RuleSnoozes: snoozes,
 	}, nil
 }
+
+func (usecase RuleSnoozeUsecase) GetRuleSnoozeById(ctx context.Context, ruleSnoozeId string) (models.RuleSnooze, error) {
+	s, err := usecase.ruleSnoozeRepository.GetSnoozeById(
+		ctx, usecase.executorFactory.NewExecutor(), ruleSnoozeId)
+	if err != nil {
+		return models.RuleSnooze{}, err
+	}
+
+	return s, nil
+}
```

Comment:
Missing securuty check

Reply:
oh damn, thank you.
I went through all the steps to add the permission check, including adding the org id on the model by joining with the parent table, only to forget adding it in the usecase in the end 🤦